octopoes/octopoes/models/ooi/email_security.py

```diff
@@ -10,6 +10,13 @@
 
 
 class DNSSPFRecord(OOI):
+    """Returns the DNS SPF record.
+
+    Functions:
+    natural_keys: returns property of the class.
+    format_reference_human_readable: returns the title text for objects on the object details page.
```

Comment:
Two things:
- This isn't necessary to document here. If we want to document them, then this should be placed in the base OOI class
- Replace (also in the other changes) "Returns" with "Represents"

```suggestion
```